### ISSUE TITLE
Investigate D3DDevice_MultiplyTransform OOVPA

### ISSUE THREAD

**Issue by RadWolfie** [Cxbx-Reloaded/XbSymbolDatabase#34]
3911 (OOVPA)
* Cell Damage (3911)
* Silent Hill 2 (3925)
* Star Wars Starfighter SE (3948)
* Azurik (4039)
* The Hulk (5344)

4034 (OOVPA)
* NFL Blitz 2002 (4242)
* RedCard 2003 (4432)
* Terminator Dawn of Fate (4721)
* Whacked! (4831)
* Dino Crisis 3 (5558)

Specific revision needs to verify:
* 4034 (Is it 3911 or 4034?)
* 4134 (4034?)

I can confirm both OOVPA are exactly the same across each revision beside 4034 and 4134 are unverified.

**Comment by jarupxx** [comment]
I can confirm Wreckless (4134) uses 4034 OOVPA.
I just can't seem to find a 4034, at least 2 titles is not contain such as Dark Summit, The Simpsons Road Rage.

**Comment by RadWolfie** [comment]
Yeah, I tried The Simpsons Road Rage title before I request for assistance. It doesn't have a full library compiled for DSound. Which is the reason why it is not on DSound's full library record of known titles.
